doc/src/compute_rattlers.rst

```diff
@@ -0,0 +1,77 @@
+.. index:: compute rattlers
+
+compute rattlers command
+========================
+
+Syntax
+""""""
+
+.. parsed-literal::
+
+   compute ID group-ID rattlers cutoff zmin ntries
+
+* ID, group-ID are documented in :doc:`compute <compute>` command
+* rattlers = style name of this compute command
+* cutoff = *type* or *radius*
+
+  .. parsed-literal::
+
+       *type* = cutoffs determined based on atom types
+       *radius* = cutoffs determined based on atom diameters (atom style sphere)
+
+* zmin = minimum coordination for a non-rattler particle
+* ntries = maximum number of iterations to remove rattlers
+
+Examples
+""""""""
+
+.. code-block:: LAMMPS
+
+   compute 1 all rattlers type 4 10
+
+Description
+"""""""""""
+
+.. versionadded:: TBD
+
+Define a compute that identifies rattlers in a system. Rattlers are
```

Comment:
I think after the 1st sentence, you should defining what a rattler is and briefly explain scenarios where it is useful to be able to identify them.  Then a 2nd paragraph could explain how this compute operates to identify them.